### PR TITLE
fix #61312 reddit.com

### LINKED ISSUE #61312: reddit.com
### Issue URL (Ads)
 [https://www.reddit.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.reddit.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.0.1
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | None
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#61312

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -235,6 +235,7 @@ xxxparodyhd.net,movgotv.com,123kmovies.net,123movies.*,bluemovies.co,f5movies.to
 icutlink.com##form[method="post"] > br
 ||safelinku.com/js/web-script.js
 ||goaloo.*/mn/*.gif
+reddit.com##article[id] > a[href^="https://alb.reddit.com/"] ~ *
```
Comment (BlazDT) [inline EnglishFilter/sections/specific.txt:238]: To Mobile. On desktop I get no hit.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/specific.txt]: I was just confused

**Review by BlazDT — APPROVED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/specific.txt]: @Sergey-Lyapin @BlazDT Dou you still see it? I cannot find.

Comment (BlazDT) [reply EnglishFilter/sections/specific.txt]: In mobile sometimes.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**